### ISSUE TITLE
UX: setup stuck at "Please touch the YubiKey"

### ISSUE THREAD

**Issue by lalyos** [str4d/age-plugin-yubikey#104]
## What were you trying to do

I'm trying to follow [Configuration docs](https://github.com/str4d/age-plugin-yubikey#configuration)

## What happened

```
$ age-plugin-yubikey --version
age-plugin-yubikey 0.3.2

$ age-plugin-yubikey

⏳ Please insert the YubiKey you want to set up.
🔑 Select a YubiKey: Yubico YubiKey OTP+FIDO+CCID (Serial: 4160736)
🕳️  Select a slot for your age identity: Slot 1 (Empty)
📛 Name this identity [age identity TAG_HEX]: lly
🔤 Select a PIN policy: Once   (A PIN is required once per session, if set)
👆 Select a touch policy: Always (A physical touch is required for every decryption)
Generate new identity in slot 1? yes


Enter PIN for YubiKey with serial 4160736 (default is 123456): [hidden]
👆 Please touch the YubiKey
... hangs ...

```

Any tips what I'm missing?

## Device

- I have a YubiKey 4 nano with 4.2.6 firmware
- os: macOS Monterey v12.5.12
- I've already tried the "reset" with the official YubiKey Manager (see screenshot below)

## Screenshot

<img width="921" alt="image" src="https://user-images.githubusercontent.com/464122/210578505-d8b33384-23c8-4056-8a41-6521ec7b2dee.png">

**Comment by str4d** [comment]
First, the obvious question: did you touch the YubiKey? 😁 (That is what it should be hanging for.)

Assuming you did, was the YubiKey flashing when you touched it? Did it keep flashing afterwards?

Could you try checking out the `main` branch of this repo and try using it? (`cargo run`)

**Comment by lalyos** [comment]
fair enough ... Yes indeed I touched it. Tried to pull-out and reinsert into the usb slot several times ...
